Source/Plugins/Processing/ProcessingFilters/MinNeighbors.cpp

```diff
@@ -184,7 +184,8 @@ void MinNeighbors::dataCheck()
   getDataContainerArray()->validateNumberOfTuples(this, dataArrayPaths);
 
   // Throw a warning to inform the user that the neighbor list arrays could be deleted by this filter
-  QString ss = QObject::tr("If this filter changes the selected feature ids array, all NeighborList arrays will be deleted.");
+  QString featureIdsPath = getFeatureIdsArrayPath().getDataContainerName() + "/" + getFeatureIdsArrayPath().getAttributeMatrixName() + "/" + getFeatureIdsArrayPath().getDataArrayName();
+  QString ss = QObject::tr("If this filter changes the selected feature ids array located at '%1', all NeighborList arrays will be deleted.").arg(featureIdsPath);
```

Comment:
I still think the wording should be ...., all arrays of type NeighborList will be Deleted.  The way you have it now it makes me looks for arrays _named_ NeighborList.

If you really wanted to be slick, you could walk all the attribute arrays in the Attribute matrix, find the ones that are of type NeighborList, and specifically mention those in the warning message